src/ScoutingMain.elm

```diff
@@ -246,11 +246,16 @@ update msg model =
 
                 verifier : Bool
                 verifier =
-                    (not <| List.member matchError [ Err "No such match", Err "Invalid match number" ])
-                        && stationError
+                    stationError
                         /= Err "No station"
                         && (not << String.isEmpty << .scouterName << .teamData) model
                         || List.member model.teamData.scouterName [ "Itamar", "tom", "hadar", "shira" ]
+                        && (if model.teamData.teamEdit then
+                                matchError /= Err "Invalid match number"
+
+                            else
+                                not <| List.member matchError [ Err "No such match", Err "Invalid match number" ]
+                           )
```

Comment:
Is all these conditions really needed?
If you're done testing the errors, shouldn't this be just
```elm
List.all Result.Extra.isOk [ stationError, matchError ] -- You can add more `Result String Int`s here as needed
```
?